raiden/tests/scenarios/mfee3_only_imbalance_fees.yaml

```diff
@@ -54,6 +55,7 @@ scenario:
             - wait_blocks: 2
             # Check that the PFS returns a path from 0 to 3
             - transfer: {from: 0, to: 2, amount: 500_000_000_000_000_000, expected_http_status: 200}
+            - wait_blocks: 1  # Wait for balances being up to date
```

Comment:
@Dominik1999 you had told me that you had seen problems with waiting 10 seconds here in the past. Is 1 blocks enough? Perhaps turn it to seconds and let's find a common number that works everywhere? In other scenarios I have even seen this be 100 seconds: 

https://github.com/raiden-network/raiden/blob/88664b202291c4d594e7cebf55d91cd8f41cf862/raiden/tests/scenarios/bf1_basic_functionality.yaml#L230

Reply:
Do we have a number that works everywhere? Our adding wait seems to be not the most exact science.

Reply:
One block was sufficient for all cases I tried and I would assume a short time (1-5 seconds) should be sufficient to have them succeed with a high probability. But since scenarios are slow to run, it's hard to try a few thousand cases to get any reliable measures.

Reply:
but then let's go for that here and adjust if it fails?